### PR TITLE
Upgrade to netcoreapp22

### PR DESCRIPTION
**Pull request by ryanbrandenburg**

### CODE REVIEW

File: build/dependencies.props [hunk 1]
```diff
@@ -4,39 +4,40 @@
...
-    <MicrosoftAspNetCoreAuthenticationCorePackageVersion>2.2.0-preview1-34184</MicrosoftAspNetCoreAuthenticationCorePackageVersion>
-    <MicrosoftAspNetCoreBenchmarkRunnerSourcesPackageVersion>2.2.0-preview1-34184</MicrosoftAspNetCoreBenchmarkRunnerSourcesPackageVersion>
-    <MicrosoftAspNetCoreHostingAbstractionsPackageVersion>2.2.0-preview1-34184</MicrosoftAspNetCoreHostingAbstractionsPackageVersion>
-    <MicrosoftAspNetCoreHostingPackageVersion>2.2.0-preview1-34184</MicrosoftAspNetCoreHostingPackageVersion>
-    <MicrosoftAspNetCoreHttpExtensionsPackageVersion>2.2.0-preview1-34184</MicrosoftAspNetCoreHttpExtensionsPackageVersion>
-    <MicrosoftAspNetCoreHttpOverridesPackageVersion>2.2.0-preview1-34184</MicrosoftAspNetCoreHttpOverridesPackageVersion>
-    <MicrosoftAspNetCoreHttpPackageVersion>2.2.0-preview1-34184</MicrosoftAspNetCoreHttpPackageVersion>
-    <MicrosoftAspNetCoreHttpSysSourcesPackageVersion>2.2.0-preview1-34184</MicrosoftAspNetCoreHttpSysSourcesPackageVersion>
-    <MicrosoftAspNetCoreServerIntegrationTestingPackageVersion>0.6.0-a-preview1-portableFlag-17022</MicrosoftAspNetCoreServerIntegrationTestingPackageVersion>
```
Comment (Tratcher) [inline build/dependencies.props]: Has this change made it through the CI yet?

File: Directory.Build.targets [hunk 2]
```diff
@@ -3,8 +3,11 @@
     <RuntimeFrameworkVersion Condition=" '$(TargetFramework)' == 'netcoreapp2.0' ">$(MicrosoftNETCoreApp20PackageVersion)</RuntimeFrameworkVersion>
     <RuntimeFrameworkVersion Condition=" '$(TargetFramework)' == 'netcoreapp2.1' ">$(MicrosoftNETCoreApp21PackageVersion)</RuntimeFrameworkVersion>
     <RuntimeFrameworkVersion Condition=" '$(TargetFramework)' == 'netcoreapp2.2' ">$(MicrosoftNETCoreApp22PackageVersion)</RuntimeFrameworkVersion>
+    <RuntimeFrameworkVersion Condition=" '$(TargetFramework)' == 'netcoreapp2.2' ">$(MicrosoftNETCoreApp22PackageVersion)</RuntimeFrameworkVersion>
```
Comment (jkotalik) [inline Directory.Build.targets]: Huh?

File: build/dependencies.props (hunk 1 above)
Comment (Tratcher) [reply build/dependencies.props]: https://github.com/aspnet/Hosting/pull/1420 which is in http://aspnetci/viewLog.html?buildId=469101&buildTypeId=UniverseCoherence&tab=buildChangesDiv which made it into 34215 so you should be good.

File: Directory.Build.targets (hunk 2 above)
Comment (ryanbrandenburg) [reply Directory.Build.targets]: Nice catch, looks like I accidentally ran portions of the script twice. I'll keep an eye out for similar mistakes on the other PRs.

**Pull request merged**